app/build.gradle

```diff
@@ -0,0 +1,33 @@
+apply plugin: 'com.android.application'
+
+apply plugin: 'kotlin-android'
+
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 28
+    defaultConfig {
+        applicationId "com.lavanya.graphqlandroid"
```

Comment:
Maybe change the package name to ```org.aerogear.graphql.androidsample```?

@wtrocki what do you say?

Reply:
I think Android is not needed in the packege `org.aerogear.graphql.sample` is good enough